modules/periphery/3.0.1/MODULE.bazel

```diff
@@ -0,0 +1,28 @@
+module(
+    name = "periphery",
+    version = "3.0.1",
+    compatibility_level = 1,
+)
+
+# Bazel dependencies
+bazel_dep(name = "rules_swift", version = "2.3.0")
+bazel_dep(name = "rules_apple", version = "3.16.0")
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+bazel_dep(name = "apple_support", version = "1.17.1")
+
+# Swift dependencies
+bazel_dep(name = "aexml", version = "4.7.0")
+bazel_dep(name = "swift_argument_parser", version = "1.5.0")
+bazel_dep(name = "swift-filename-matcher", version = "2.0.0")
+bazel_dep(name = "swift-indexstore", version = "0.3.0")
+bazel_dep(name = "swift-syntax", version = "600.0.1")
+bazel_dep(name = "swift-system", version = "1.4.0")
+bazel_dep(name = "pathkit", version = "1.0.1")
+bazel_dep(name = "xcodeproj", version = "8.25.0")
+bazel_dep(name = "yams", version = "5.1.3")
+
+# Generated repo
+use_repo(use_extension("//bazel:generated.bzl", "generated"), "periphery_generated")
```

Comment:
Just FYI, no need to act: While this shorthand form is supported by Bazel on a semantic level, tooling including `bazel mod tidy` will be confused by it.